src/main/java/com/odysseusinc/arachne/portal/service/BaseUserService.java

```diff
@@ -81,6 +81,8 @@
     void remove(Long id)
             throws ValidationException, UserNotFoundException, NotExistException, IOException, SolrServerException;
 
+    List<U> createAll(final @NotNull BulkUsersRegistrationDTO bulkUsersDto) throws PasswordValidationException, ValidationException;
```

Comment:
Do not use controller level DTOs in services. Either create service level or pass as separate params and converted models (get a look at other places)